Gemfile

```diff
@@ -159,6 +159,9 @@ group :test do
   # Clean state in-between tests which modify the DB
   gem "database_cleaner"
 
+  # Create a temporary table-backed ActiveRecord model
```

Comment:
Would you move this gem to the development/test block?

Reply:
Addressed 👍 
https://github.com/brave-intl/publishers/pull/1396/commits/1f3709dc373273bdbf7ceba0b395aad18c7b1313